katunold/ekalaamu
# #167128050 user login

#### What does this PR do?
- add user login UI and implementation
- add links to landing page and signup form
- refactor folder structure
#### Description of Task to be completed?
- allow users to login into ekalaamu if they're registered
#### How should this be manually tested?
1. clone this repo and check into the branch `ft-user-login-167128050` 
2. start the server `npm start` 
3. navigate to the route `auth/login`
